content/applications/websites/livechat.rst

```diff
@@ -267,8 +298,8 @@ screen, no matter where they are in the database. They can participate in conver
 leaving their current location.
 
 .. tip::
-   Conversations can also be accessed by clicking the :guilabel:`Conversations` icon in the
-   menu bar.
+   Conversations can also be accessed by clicking the :guilabel:`Conversations` icon in the menu
+   bar.
```

Comment:
I think this could use some more clarification, and perhaps the GUI might not be applicable here because it's just an icon -- and the name might not be present. But, if it is, feel free to include. Also, in addition to the clarification, you may want to consider adding an icon to this portion.